### DISCUSSION TITLE
CUYUNIVERSITY Semester 4 - SESI9 - Validasi User Dashboard | Next JS 2023

### DISCUSSION THREAD

**Discussion by AndisDev** [deaafrizal/forum#45] [Troubleshooting Room] [answered]
Saya Masih terkendala di bagian middleware

sudah saya compare code nya namun masih blum solve

![image](https://github.com/deaaprizal/forum/assets/145734763/3f2e67c4-5d80-4c96-84cc-2eeae6dda6ac)

berikut code untuk 
middleware.js 
![image](https://github.com/deaaprizal/forum/assets/145734763/34b583b3-6712-4806-b2f4-6bb7f5366ccd)
route.js
![image](https://github.com/deaaprizal/forum/assets/145734763/7196ff2c-0c01-4940-a766-681842b39381)

error di terminal 
![image](https://github.com/deaaprizal/forum/assets/145734763/f7fbc405-19e5-48aa-91ba-9adb383a5cbf)

Mohon di bantu terkait error tersebut
Terima kasih

**Comment by andikaalakate** [comment, marked as answer]
Coba next auth nya diinstall lagi, update

**Reply by andikaalakate** [reply]
Atau coba lihat di .env nya, apakah ada typo di NEXTAUTH_SECRET nya

**Reply by AndisDev** [reply]
thanks kak 

ternyata harus saya install lagi next auth nya 

sudah working sekarang

**Reply by andikaalakate** [reply]
iyaa, sama-sama

**Reply by deaafrizal** [reply]
thank u @andikaalakate udah bantu!
